### DISCUSSION TITLE
Displaying pinned comments on the main feed?

### DISCUSSION THREAD

**Discussion by semihmasat** [usememos/memos#3964] [Ideas]
Hi,

Firstly i want to thank everyone involved in this app. I am using this on my home server as a knowledge database. I know there are other tools for that but this format works much better for me. It help me to take a lot of notes, screenshots and links and combine them easily. 

Anyway, as i take learning notes, i also return to them and update them. I also like to ask questions to my future self and answer them later with a comment. And i think it would be great to be able to show pinned comments on the feed as well. Like linkedin does. 

I tried to check the code to see if i can do it myself but i'm not that good enough.
Thank you.

**Comment by RoccoSmit** [comment]
Which feed are you referring to as `main`

You can currently pin memos to the top of your home feed

pin button:
![image](https://github.com/user-attachments/assets/57dba057-080f-486f-89f0-4254eca8d950)

pinned memo:
![image](https://github.com/user-attachments/assets/1f3a8de4-5541-4da9-bf73-40cca94d2aef)

**Reply by semihmasat** [reply]
Thank you for your reply, yes i meant the home. 
These pins are the memo itself but not the comments.

What i am after is seeing the comments of a memo on the home feed. ( or other feeds )

![Screenshot 2024-09-29 at 12 20 58](https://github.com/user-attachments/assets/fa54047c-84b5-4ca9-abe2-35ae0ab0ba52)
